src/vs/base/browser/ui/button/button.ts

```diff
@@ -234,6 +219,31 @@ export class Button extends Disposable implements IButton {
 	}
 }
 
+export class Button extends GenericButton implements IButton {
+
+	constructor(container: HTMLElement, options?: IButtonOptions) {
+		super(container, options);
+
+		this.element.classList.add('monaco-button');
+		this.element.tabIndex = 0;
+		this.element.setAttribute('role', 'button');
+	}
+
+	set label(value: string) {
+		this.element.classList.add('monaco-text-button');
+		if (this.options.supportIcons) {
+			reset(this.element, ...renderLabelWithIcons(value));
+		} else {
+			this.element.textContent = value;
+		}
+		if (typeof this.options.title === 'string') {
+			this.element.title = this.options.title;
+		} else if (this.options.title) {
+			this.element.title = value;
+		}
+	}
```

Comment:
This is the duplicate/duplicated setter referenced in my previous comment.

Reply:
Hi @ferdnyc,
yes, you are right, this code block is duplicated as well and could be reduced.